app/javascript/components/review-summary-card.vue

```diff
@@ -0,0 +1,43 @@
+<script setup lang="ts">
+
+type Props = {
+  count: string,
+  average:string
+}
+
+defineProps<Props>();
+
+</script>
+
+<template>
+  <div class="flex items-center justify-start gap-4">
+    <div>
+      <p class="text-6xl text-zinc-800">
+        {{ parseFloat(average).toFixed(1) }}
+      </p>
+    </div>
+    <div class="flex flex-col">
+      <div class="flex gap-3">
+        <div
+          v-for="n in 5"
+          :key="n"
+        >
+          <inline-svg
+            v-if="n <= average"
```

Comment:
🚫 **[tsc]** <2365> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Operator '<=' cannot be applied to types 'number' and 'string'.